### DISCUSSION TITLE
Is there a plan to encrypt .tfvars files in CICD repos?

### DISCUSSION THREAD

**Discussion by v-wan** [GoogleCloudPlatform/cloud-foundation-fabric#1862] [General]
Interested about the feasibility of using KMS + SOPS to encrypt the .tfvars files so it does not get revealed in the stage CICD repositories.

**Comment by ludoo** [comment]
Why would you want to encrypt tfvars? They do not typically contain sensitive data, except possibly for the VPN secrets but that would be better handled at the VPN module level by using Secret Manager.

**Reply by v-wan** [reply]
Thanks for answering, we are trying to keep the repo security as tight as possible, like hiding values such as billing ID. For now we are using the KMS keys in the security stage for encryption.
